backend/server.js

```diff
@@ -130,12 +131,24 @@ const addRepoToCheck = (repoURL) => {
   }
 };
 
-const cronJob = new cron.CronJob("* * * * *", () => {
-  console.log("Checking for new GFIs...");
-  repoToCheck.forEach((repoURL) => {
-    sendGFIEmail(repoURL);
-    console.log(`Trying to send an email for ${repoURL}.`);
-  });
+const cronJob = new cron.CronJob('* * * * *', async () => {
+  
+  console.log('Checking for new GFIs...');
+  try {
+    if (repoToCheck.length == 0) {
+      let repoData = await Subscription.find({});
+      repoData = [...new Set(repoData.map(item => item.repoUrl))];
```

Comment:
This needs to be changed to `repoURL` instead of `repoUrl`

Reply:
Ok, I'll change it